deployer/templates/curator.yaml

```diff
@@ -134,9 +138,17 @@ parameters:
   value: "/etc/curator/keys/ca"
 -
   description: "(Deprecated) The default number of days that logs will be retained for a project if an index management configuration for it is not provided"
-  name: CURATOR_DEFAULT_DAYS
+  name: CURATOR_DEFAULT_DELETE_DAYS
   value: "30"
 -
+  description: "(Deprecated) The default number of days that logs will be left open for a project if an index management configuration for it is not provided"
```

Comment:
this was more following what we already had, we can probably remove this one and the default for delete TBH